image/helpers.go

```diff
@@ -213,19 +213,26 @@ type DownloadOptions struct {
 	TargetDir string
 	Channel   string
 	CohortKey string
+	Basename  string
```

Comment:
`BaseName`?

Reply:
I could take a page from `path/filepath` and just call it `Base` everywhere

Reply:
that won't be confusing at all

Reply:
to be clear: I was agreeing with you until it struck me that it would be confusing given we have something called a base... (elaborated the point on IRC)

Reply:
to be clear basename is usually one word in our codebase